pkg/pack.sh

```diff
@@ -44,7 +71,9 @@ if [ "$(uname -s)" == "Darwin" ]; then
     __targets_param="TargetsOSX=true"
 else
     __targets_param="TargetsLinux=true"
-    init_distro_name
+    if [ -z $__distro_rid ]; then
```

Comment:
If the pack.sh is always invoked by dotnet-build-tools, the __distro_rid will always be set. I considered the situation where the pack.sh is called directly without --rid option. However, in your opinion, I will modify this part in the next patch. Thank you.